app/Services/Twilio.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ivan
+ * Date: 04.06.16
+ * Time: 15:24
+ */
+
+namespace App\Services;
+
+use App\Models\Countries;
+use App\Models\PhoneNumbers;
+use Services_Twilio;
+use Exception;
+use App\Exceptions\NoTwilioAvailableNumberException;
+use App\Exceptions\NoTwilioIncomingNumberException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class Twilio
+{
+
+    const THANKING_MESSAGE_DURATION = 120; //60 * 2 seconds
+
+    const DEFAULT_APPLICATION_SID = 'AP6452316e22874c8e9f4c3425832b0b28';
+
+    const SHORT_CONVERSATION_MESSAGE = 'Thank you very much';
```

Comment:
I have done it just to save time. It's better to store messages in some resource files, that can be translated in future and easily find, if it should be changed